test/playlist.test.js

```diff
@@ -633,6 +633,29 @@ QUnit.test('seekable and playlistEnd use available sync points for calculating',
     assert.equal(playlistEnd, 148.5, 'playlist end at the last segment end');
   });
 
+QUnit.module('Playlist isAes and isFmp4');
+
+QUnit.test('determine if playlist is an AES encrypted HLS stream', function(assert) {
+  let media = '#EXTM3U\n' +
+              '#EXT-X-TARGETDURATION:15\n' +
+              '#EXT-X-KEY:METHOD=AES-128,URI="http://example.com/keys/key.php"\n' +
+              '#EXTINF:2.833,\n' +
+              'http://example.com/000001.ts\n' +
+              '#EXT-X-ENDLIST\n';
+
+  assert.ok(Playlist.isAes(media), 'media is an AES encrypted HLS stream');
```

Comment:
You either need to pass in an object representation of a playlist like in the test above or create a loader like the test 2 below and respond to the xhr request with this string. These tests should be failing, but passed in traviscli (and probably locally for you) because of the overriding of the `isAes` and `isFmp4` methods you did in `master-playlist-controller.test.js`